Maps.md

```diff
@@ -0,0 +1,21 @@
+This is a general guide about Maps for all FullScreenShenanigans projects. This guide will explain the concept of Maps, Areas, and Locations within a game and how they're connected.
+
+### Table of Contents
+1. [MapsCreatr](#mapscreatr)
+2. [AreaSpawnr](#areaspawnr)
+
+# MapsCreatr
+
+MapsCreatr is a module that handles the storage of Maps, Areas, and Locations. 
+
+A Map is a collection of Areas. An Area is a 2D space with predefined  `x` and `y` boundaries and Things. Locations are entry points within an Area, where an entry point is where the player ends up after transitioning to a new Area.
+
+Generally in a FullScreenShenanigans project, each Map information is stored in its own .js file, outlining all Areas and Locations within that map. When the game is started, MapsCreatr loads all this data, goes through every Map and uses ObjectMakr to create a new Map object, and finally stores it for later reference.
```

Comment:
> goes through every Map and uses ObjectMakr to create a new Map object

This is unclear and I don't know what you mean by it. Say a new Map object is created using the JSON description.

*(rephrase that last sentence I wrote)*